web/src/engine/osk/src/input/gestures/browser/pendingMultiTap.ts

```diff
@@ -1,15 +1,15 @@
 import { Codes, KeyEvent } from '@keymanapp/keyboard-processor';
 import { type KeyElement } from '../../../keyElement.js';
 import VisualKeyboard from '../../../visualKeyboard.js';
-import PendingGesture from '../pendingGesture.interface.js';
+// import PendingGesture from '../pendingGesture.interface.js';
 
 export enum PendingMultiTapState { Waiting, Realized, Cancelled };
 /**
  * Implements the multi-tap gesture, which is a series of taps on a single key
  * (based on key id substring in the case of the shift key), within a
  * specified timeout period.
  */
-export default class PendingMultiTap implements PendingGesture {
+export default class PendingMultiTap /*implements PendingGesture*/ {
```

Comment:
why commented out?

Reply:
`PendingGesture` has been removed from the code base.  Just... forgot to go back and polish this ol' reference up.